js/dominion.js

```diff
@@ -833,7 +833,8 @@
 
   $(function ()
     {
-      var $advancedSearch = $('body.informationobject.browse');
+      // Find search for if on an appropriate page
+      var $advancedSearch = $('body.informationobject.browse').add('body.search.descriptionUpdates');
```

Comment:
I tried using the command-separated initially and for some reason couldn't get it to work. But I'll do a quick re-try.